tests/test_events.py

```diff
@@ -44,6 +45,11 @@ def run_test(client, base_ddo_url, events_instance, flags):
     assert published_ddo["id"] == did
     assert published_ddo["metadata"]["name"] == "Updated ddo by event"
 
+    send_set_metadata_state_tx(_ddo,test_account1)
```

Comment:
there is no need to put this in run_test, it will retest the same thing too many times. Please create a separate test.